packages/terra-text/src/terra-dev-site/doc/example/TextFontSizes.jsx

```diff
@@ -1,6 +1,12 @@
 import React from 'react';
 import Text from 'terra-text';
 
+// The `fontSize` prop can be used set the `font-size` on any `<Text />` component. The `fontSize` prop accepts one of the following numbers:
```

Comment:
I think these comments would be better placed as descriptions in for the examples in the Mdx file.